### ISSUE TITLE
When I create route from Env. Log the route will never return response

### ISSUE THREAD

**Issue by usrbowe** [mockoon/mockoon#229]
**Describe the bug**
Requesting mock route, never resolves. Keep waiting for response. If I create same route manually it works well.

**To Reproduce**
1. Turn On proxy mod
2. Wait for real request to be logged in Environmental Log
3. Create a mock route from env. log
4. Try to request the mocked route
5. It just keeps waiting for response, never resolves.

**Expected behaviour**
When request the mock route, it should return response.

**Mockoon version:**
`1.7.0`

**OS / OS version:**
Mac OS `10.14`

**Comment by 255kb** [comment]
@usrbowe I tested and I get a JSON parsing error due to the fact that the route is automatically created with a lower case "content-type". Apart from this bug I see no other problem.
Could you please check if it's the same bug and if not, give some sample data to allow me reproduce it?
Thank you

**Comment by usrbowe** [comment]
Oh, headers might the issue. I'm not seeing any JSON error in Mockoon when request the route, it just stays at waiting for response. So can't confirm your case.

But I did notice I'm not able to remove any headers, when I delete them and switch tabs to response a go back to Headers, it shows the original again:
![Screen Recording 2020-01-21 at 09 16 AM](https://user-images.githubusercontent.com/5339061/72767465-eff20800-3c2e-11ea-84a9-0e22e65fd634.gif)

So possibly I do have issue with some headers, which breaks the response, but I'm not able to remove them :) :
```js
content-encoding: 'gzip',
content-type: 'application/json; charset=utf-8',
connection: 'close',
access-control-allow-origin: 'http://localhost:3018', // my server actually runs on :3002
access-control-allow-headers: 'Content-Type,Authorization,X-Requested-With,X-CSRFToken,If-None-Match-,X-API-SOURCE'
```

**Comment by wasder** [comment]
I recorded a video with this problem.
[](https://github.com/mockoon/mockoon/files/4091850/2020-01-21.18.35.30.zip)

**Comment by 255kb** [comment]
I watched the video, and I have to say, this is strange. What value do you have in the header `Access-control-allow-origin`? Could it be possible that it contains a special invisible character that crash the application? 
Because not being able to delete the headers indicates that something went very wrong like JS crashing.

Could you please share with me your `environments.json` file located in Mockoon application data folder? You can access the folder by clicking in Help menu -> Show app data folder. 
Then go to `storage`. 
Thank you

**Comment by wasder** [comment]
```
Access-control-allow-credentials: true
Access-control-allow-origin:
Connection: close
Content-length: 1195
Content-type: application/json;charset=UTF-8
Date: Mon, 27 Jan 2020 12:18:54 GMT

```
```

[ 
   { 
      "uuid":"5fa3adc0-3b7b-11ea-a70e-550e3c72350d",
      "lastMigration":9,
      "name":"Test bug",
      "endpointPrefix":"",
      "latency":0,
      "port":9393,
      "routes":[ 
         { 
            "uuid":"ba791df1-3c63-11ea-a769-c3e57168cb4b",
            "documentation":"",
            "method":"get",
            "endpoint":"np-test/2-67/api/v1.0/exchange/rates/regular/",
            "responses":[ 
               { 
                  "uuid":"ba791df0-3c63-11ea-a769-c3e57168cb4b",
                  "body":"{\"success\":true,\"data\":[{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"USD\",\"outputCurrencyName\":\"USD\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":63.05,\"buyRate\":61.0,\"cbRate\":61.9057},{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"EUR\",\"outputCurrencyName\":\"EUR\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":70.0,\"buyRate\":67.9,\"cbRate\":69.3777},{\"inputCurrency\":\"USD\",\"outputCurrency\":\"EUR\",\"outputCurrencyName\":\"EUR\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":1.0,\"buyRate\":1.0,\"cbRate\":1.0},{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"A33\",\"outputCurrencyName\":\"Палладий\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":3964.4,\"buyRate\":3644.6,\"cbRate\":5084.65},{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"A76\",\"outputCurrencyName\":\"Платина\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":1968.2,\"buyRate\":1813.3,\"cbRate\":2025.56},{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"A99\",\"outputCurrencyName\":\"Серебро\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":37.0,\"buyRate\":34.05,\"cbRate\":35.69},{\"inputCurrency\":\"RUR\",\"outputCurrency\":\"A98\",\"outputCurrencyName\":\"Золото\",\"date\":\"2020-01-21T00:00:00.000+0300\",\"saleRate\":3136.7,\"buyRate\":2886.1,\"cbRate\":3081.32}]}",
                  "latency":0,
                  "statusCode":"200",
                  "label":"",
                  "headers":[ 
                     { 
                        "key":"access-control-allow-credentials",
                        "value":"true"
                     },
                     { 
                        "key":"access-control-allow-origin",
                        "value":""
                     },
                     { 
                        "key":"connection",
                        "value":"close"
                     },
                     { 
                        "key":"content-length",
                        "value":"1195"
                     },
                     { 
                        "key":"content-type",
                        "value":"application/json;charset=UTF-8"
                     },
                     { 
                        "key":"date",
                        "value":"Tue, 21 Jan 2020 15:35:45 GMT"
                     },
                     { 
                        "key":"set-cookie",
                        "value":[ 
                           "SERVERNAMEW=rd117o00000000000000000000ffff0a307220o443; expires=Tue, 21-Jan-2020 17:35:45 GMT; path=/; Httponly; Secure",
                           "f5avraaaaaaaaaaaaaaaa_session_=HBOBKLPCFFBCAKPIFFGLNOEKOEOBDNPJBCHELPFDHPGMDHBPOOBDNHGOHGPMPOHIJGFDNGPCGKDCGPDCMKBABHAAABDLLAKPGLACKGFBFMNIDOBEPJONBELJLKFKJMNP; HttpOnly; secure",
                           "TS01ac3987=016fed35e7c60fc6f1a527bcb8d97bbc6ceb7fe31e44a055cea02b4a4fbb114f2f7c1caa49b1e00a186044bd0442e9a3625e1cfb338e4d0e84291460300c353acc9718c5c8a0f5bc72b27f0fb1026276d937992210; Path=/; Domain=.open.ru;Secure"
                        ]
                     },
                     { 
                        "key":"spantraceid",
                        "value":"28792beb69b4d90d"
                     }
                  ],
                  "filePath":"",
                  "sendFileAsBody":false,
                  "rules":[ 

                  ]
               }
            ],
            "enabled":false
         }
      ],
      "proxyMode":true,
      "proxyHost":"https://api1.open.ru/",
      "https":false,
      "cors":false,
      "headers":[ 

      ]
   }
]




```

**Comment by ishamf** [comment]
I think if there are multiple headers with the same name, it's not handled correctly. I did a quick fix in my fork here https://github.com/ishamf/mockoon/tree/fix-multi-cookies and seems to work now.

**Comment by wasder** [comment]
@ishamf 
Can you build it for MacOS, so I can test it, please?

**Comment by 255kb** [comment]
@wasder Didn't see that `set-cookie` header was an array. Indeed, header objects should be plain string. 
@ishamf I think your fix is exactly what is needed here, I didn't realize that headers could contains arrays. Would you mind opening a PR for this? If you don't have time I can copy your fix. Let me know.

**Comment by 255kb** [comment]
I was able to reproduce the bug, it's corrected and will be in v1.8.0
